docker/docker-compose.yaml

```diff
@@ -34,14 +37,10 @@ services:
     <<: *common
     command: >-
       /bin/bash -clx "
-      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/Benchmark &&
-      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/Deployment &&
-      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/Echo &&
-      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/ErrorHandling &&
-      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/Foundation &&
-      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/JSON &&
-      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/LocalDebugging/MyLambda &&
-      LAMBDA_USE_LOCAL_DEPS=true swift test --package-path Examples/Testing
+      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/APIGateway &&
+      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/AWSSDK &&
+      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/HelloWorld &&
+      LAMBDA_USE_LOCAL_DEPS=true swift build --package-path Examples/Soto
```

Comment:
Let's make this a test matrix :)

Reply:
I added a loop to automate the discovery of Examples